lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
-
+import PropTypes from 'prop-types'
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} />
         </div>
-        <span>By {/* author goes here */}</span>
+        <span>By {props.card.author}</span>
       </div>
     </div>
   );
 };
 
 // Make sure to include PropTypes.
+Card.PropTypes = {
```

Comment:
Awesome work with PropTypes! Be sure that you add in proptypes as a dependency. You can do this with `yarn add prop-types` inside of the lambdatimes folder